runtime/oti/j9nonbuilder.h

```diff
@@ -5514,7 +5514,7 @@ typedef struct J9VMThread {
 	UDATA contiguousIndexableHeaderSize;
 	UDATA discontiguousIndexableHeaderSize;
 #if defined(J9VM_ENV_DATA64)
-	UDATA isIndexableDataAddrPresent;
+	BOOLEAN isIndexableDataAddrPresent;
```

Comment:
`AuxFieldInfo29.dat` should have been updated to be consistent with this change.

However, this may cause problems for DDR accessing older system dumps, particularly for big-ending systems if `UDATA` and `BOOLEAN` have different sizes.

Reply:
I see this was reverted by #20816 (without explanation).